src/bundle.c

```diff
@@ -2502,6 +2502,50 @@ gboolean load_manifest_from_bundle(RaucBundle *bundle, RaucManifest **manifest,
 	return res;
 }
 
+static gboolean read_complete_dm_device(gchar *dev, GError **error)
+{
+	int fd = -1;
+	g_autofree void* buf = NULL;
+	ssize_t r;
+	gboolean ret = TRUE;
+
+	g_return_val_if_fail(dev != NULL, FALSE);
+	g_return_val_if_fail(error == NULL || *error == NULL, FALSE);
+
+	fd = g_open(dev, O_RDONLY | O_CLOEXEC, 0);
+	if (fd < 0) {
+		int err = errno;
+		g_set_error(error,
+				G_FILE_ERROR,
+				g_file_error_from_errno(err),
+				"Failed to open %s: %s", dev, g_strerror(err));
+		return FALSE;
+	}
+
+	lseek(fd, 0, SEEK_SET);
+
```

Comment:
This `lseek` is redundant, as newly opened files are always at offset 0.

Reply:
removed